samples/apps/research-assistant-react/backend/run_server.sh

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This file is present in the repo but never referenced in the readme. Delete or refer

Reply:
Which file?

Reply:
`run_server.sh`

Reply:
Definitely do not delete that file. It greatly simplifies getting the environment set up. Better to document it I guess.